.gitignore

```diff
@@ -7,3 +7,4 @@
 .vscode
 .DS_Store
 *.jar
+!edge-custom-policy-java-hello.jar
```

Comment:
Can you help me understand why we need this?

Reply:
I will be removing this ! as its now handled in the pipeline !